src/modules/launcher/Plugins/Microsoft.Plugin.Program/Storage/Win32ProgramRepository.cs

```diff
@@ -189,6 +190,7 @@ private Programs.Win32Program GetAppWithSameLnkResolvedPath(string lnkResolvedPa
         {
             foreach (Programs.Win32Program app in Items)
             {
+                // Using CurrentCulture since this is user facing
                 if (lnkResolvedPath.ToLower(CultureInfo.CurrentCulture).Equals(app.LnkResolvedPath, StringComparison.CurrentCultureIgnoreCase))
```

Comment:
`lnkResolvedPath` is a physical path and not a user-facing string. The intent here is to check if two paths are exact. This should be an ordinal culture.